src/patternfly/components/Progress/progress-indicator.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="pf-c-progress__indicator{{#if progress-indicator--modifier}} {{progress-indicator--modifier}}{{/if}}" 
-  style="width: {{progress__value}}%;"
+style="width: {{~#if progress__width}}{{progress__width}}{{else}}{{progress__value}}{{/if}}%;"
```

Comment:
Keep a close eye on the handlebar spacing - the two spaces at the beginning of the second line should not be removed.

Reply:
Line break removed.